clients/extension/src/pages/popup/pages/import/index.tsx

```
@@ -234,8 +237,11 @@ const Component = () => {
       })
     }
   }
+
   // eslint-disable-next-line react-hooks/exhaustive-deps
-  useEffect(componentDidMount, [])
+  useEffect(() => {
+    componentDidMount()
+  }, [walletCore])
```

ESLint

[error] React Hook useEffect has a missing dependency: 'componentDidMount'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)